app/models/provider_relationship_permissions.rb

```diff
@@ -0,0 +1,6 @@
+class ProviderRelationshipPermissions < ApplicationRecord
```

Comment:
Should we namespace these under `ProviderInterface`?

Reply:
Updated here https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2112/commits/c7d0a60b353a1fa1dda26b0f441708348d21885d